src/components/Pressable.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
You can make a new directory called `Pressable` and move the file with props along with this one there. Then you can add an `index` file exporting what's necessary.

Reply:
Done